docs/sources/operator/deploy-agent-operator-resources.md

```diff
@@ -53,7 +53,12 @@ To deploy the `GrafanaAgent` resource:
       labels:
         app: grafana-agent
     spec:
-      image: grafana/agent:v0.32.0
+      image: grafana/agent:v0.32.1
```

Comment:
this merge added some stuff, please sanity check

Reply:
Weird, look like it's not in the release branch, so we should probably take it out